app/controllers/users_controller.rb

```diff
@@ -0,0 +1,68 @@
+class UsersController < ApplicationController
+
+  before_action :authorize_admin_token,except: [:create]
```

Comment:
use appropriate names.
change 'authorize_admin_token' to 'authorize_admin_access'